packages/client/hmi-client/src/components/workflow/tera-simulate-chart.vue

```diff
@@ -214,11 +214,16 @@ const renderGraph = () => {
 			.forEach((run, runIdx) => {
 				const dataset = {
 					data: run.map(
-						(datum: { [key: string]: number }) => datum[variable] // - runResults[selectedRun.value][timeIdx][code]
+						// - runResults[selectedRun.value][timeIdx][code]
+						(datum: { [key: string]: number }) =>
+							// return datum[variable];
+							({
+								y: +datum[variable],
```

Comment:
Why the plus operator?

Reply:
coerce numbers, they are strings for some reason.